server/pulp/server/managers/content/query.py

```diff
@@ -165,23 +166,25 @@ def get_multiple_units_by_keys_dicts(self, content_type, unit_keys_dicts, model_
         """
         Look up multiple content units in the collection for the given content
         type collection that match the list of keys dictionaries.
-        @param content_type: unique id of content collection
-        @type content_type: str
-        @param unit_keys_dicts: list of dictionaries whose key, value pairs can
+        :param content_type: unique id of content collection
+        :type content_type: str
+        :param unit_keys_dicts: list of dictionaries whose key, value pairs can
                                 uniquely identify a content unit
-        @type unit_keys_dicts: list of dict's
-        @param model_fields: fields of each content unit to report,
+        :type unit_keys_dicts: list of dict's
+        :param model_fields: fields of each content unit to report,
                              None means all fields
-        @type model_fields: None or list of str's
-        @return: tuple of content units found in the content type collection
+        :type model_fields: None or list of str's
+        :return: tuple of content units found in the content type collection
                  that match the given unit keys dictionaries
-        @rtype: (possibly empty) tuple of dict's
-        @raise ValueError if any of the keys dictionaries are invalid
+        :rtype: (possibly empty) tuple of dict's
+        :raise ValueError if any of the keys dictionaries are invalid
```

Comment:
should be ``:raises ValueError:``

http://sphinx-doc.org/domains.html#info-field-lists